include/handler/HumidityHandler.hpp

```diff
@@ -0,0 +1,25 @@
+/*
+*   HumidityHandler.hpp
+*   ----------------------
+*   Created on: 2023/09/26
+*   Author: Lankow
+*/
+#ifndef HUMIDITY_HANDLER_HPP
+#define HUMIDITY_HANDLER_HPP
+
+#include "Component.hpp"
+
+class HumidityHandler : public Component
+{
+    private:
+    uint8_t m_operatedPin;
+    uint8_t m_handlerId;
+
+    public:
+    HumidityHandler(uint8_t pin);
+    
+    void handleHumidity();
```

Comment:
to też jest kwestia indywidualnego codestyle'u, ale wg mnie warto do metod które override;ujemy dodawać `override`. Wtedy czytając kod już wiesz, że to nadpisuje coś z bazy